### PR TITLE
refactor(ci): add more args option to logictests

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

add more args option to logictests

```
➜ python main.py -h
usage: main.py [-h] [--disable-mysql-test] [--disable-http-test] [--disable-clickhouse-test] [--test-runs [TEST_RUNS]] [--skip SKIP [SKIP ...]] [--skip-dir SKIP_DIR [SKIP_DIR ...]]
               [--run-dir RUN_DIR [RUN_DIR ...]]
               [pattern ...]

databend sqlogical tests

positional arguments:
  pattern               Optional test case name regex

options:
  -h, --help            show this help message and exit
  --disable-mysql-test  Disable mysql handler test
  --disable-http-test   Disable http handler test
  --disable-clickhouse-test
                        Disable clickhouse handler test
  --test-runs [TEST_RUNS]
                        Run each test many times (useful for e.g. flaky check)
  --skip SKIP [SKIP ...]
                        Skip these tests via case name regex
  --skip-dir SKIP_DIR [SKIP_DIR ...]
                        Skip all these tests in the dir
  --run-dir RUN_DIR [RUN_DIR ...]
                        Only run these tests in the dir
```

Fixes #issue

### CODE REVIEW

**Comment by ZeaLoVe** [comment]
Add some examples to the readme.md?

**Comment by sundy-li** [comment]
> Add some examples to the readme.md?

`python main.py -h ` will cover, this pr did not change the envs.

File: tests/logictest/logictest.py [hunk 1]
```diff
@@ -300,7 +309,8 @@ def execute(self):
                     f"Run query without session every statements, suite file path:{file_path}"
                 )
                 for state in get_statements(file_path, suite_name):
-                    self.execute_statement(state)
+                    for i in range(0, self.args.test_runs):
```
Comment (ZeaLoVe) [inline tests/logictest/logictest.py:312]: test_runs should move to function https://github.com/datafuselabs/databend/blob/e14a82a0f44369c9cb2ec347fdcdd22baa832b25/tests/logictest/logictest.py#L315-L332   

One by one execute is rarely used, usually use batch_execute.

Comment (ZeaLoVe) [reply tests/logictest/logictest.py:312]: batch_execute is implement in handlers , but call execute_statement finally.

**Review by ZeaLoVe — APPROVED** [review]

**Pull request merged**